### PR TITLE
init: react template app

### PR DESCRIPTION
**Pull request by vaibhav0806**
- add React native template app

### CODE REVIEW

File: App.js
Comment (Jovian-Dsouza) [inline App.js:1]: Api key should be in env file

File: src/SignIn.tsx
Comment (Jovian-Dsouza) [inline src/SignIn.tsx:1]: google auth secrets should be in env

File: package.json
Comment (Jovian-Dsouza) [inline package.json:1]: Could you use the latest version of okto sdk https://www.npmjs.com/package/okto-sdk-react-native

**Review by Jovian-Dsouza — CHANGES_REQUESTED** [review]
Please remove sensitive api keys , add sample env file, and update the sdk version

**Comment by vaibhav0806** [comment]
Hey @Jovian-Dsouza All the request changes have been made, please check and let me know if more are to be made.

**Review by Jovian-Dsouza — APPROVED** [review]
Changes look as expected

**Pull request merged**